wandb/sdk/data_types/table.py

```diff
@@ -640,16 +730,33 @@ def to_json(self, run_or_artifact):
         json_dict = super().to_json(run_or_artifact)
 
         if isinstance(run_or_artifact, wandb.wandb_sdk.wandb_run.Run):
+            _type = "table-file"
+            if self.log_mode == "INCREMENTAL":
+                json_dict.update(
+                    {
+                        "increment_num": self._increment_num,
+                        "previous_increments_paths": self._previous_increments_paths,
+                    }
+                )
+                _type = "incremental-table-file"
+
             json_dict.update(
                 {
-                    "_type": "table-file",
+                    "_type": _type,
                     "ncols": len(self.columns),
                     "nrows": len(self.data),
                     "log_mode": self.log_mode,
                 }
             )
 
         elif isinstance(run_or_artifact, wandb.Artifact):
+            if self.log_mode == "INCREMENTAL":
+                json_dict.update(
+                    {
+                        "increment_num": self._increment_num,
+                        "previous_increments_paths": self._previous_increments_paths,
+                    }
+                )
```

Comment:
This is the same as in the Run branch of the conditional, right? This part should be extracted and done separately to avoid duplicating it.